### PR TITLE
Fixed incorrect docs

### PR DESCRIPTION
**Pull request by currentoor**

### CODE REVIEW

File: src/main/com/fulcrologic/fulcro/ui_state_machines.cljc [hunk 1]
```diff
@@ -1059,10 +1059,10 @@
 
   `::comp/component-class` - The defsc name of the component to use for normalization and query. Only needed if the
     actor was not declared using a Fulcro component or component class.
-  `::uism/post-event`:: An event to send when the load is done (instead of calling a mutation)
-  `::uism/post-event-params`:: Extra parameters to send as event-data on the post-event.
-  `::uism/fallback-event`:: The event to send if the load triggers a fallback.
-  `::uism/fallback-event-params`:: Extra parameters to send as event-data on a fallback.
+  `::uism/ok-event`:: An event to send when the load is done (instead of calling a mutation)
```
Comment (awkay) [inline src/main/com/fulcrologic/fulcro/ui_state_machines.cljc]: you sure??? I thought I kept the old events on load...

Comment (currentoor) [reply src/main/com/fulcrologic/fulcro/ui_state_machines.cljc]: 100% this docstring is the only place that mentions `::uism/post-event` etc. Also both loads call this function
https://github.com/fulcrologic/fulcro/blob/develop/src/main/com/fulcrologic/fulcro/ui_state_machines.cljc#L1000

If you recall I suggested loads and remote mutation mutations have the same post event names to unify the API. I'm glad you did this :smile:

Comment (awkay) [reply src/main/com/fulcrologic/fulcro/ui_state_machines.cljc]: Could you also check the PORTING guide file and Dev Guide???

Comment (currentoor) [reply src/main/com/fulcrologic/fulcro/ui_state_machines.cljc]: Oh right I forgot about those, I'll update.

Comment (currentoor) [reply src/main/com/fulcrologic/fulcro/ui_state_machines.cljc]: Both of those places were already uptodate.

**Pull request merged**